index.html

```diff
@@ -1037,6 +1042,10 @@ <h4>
           <h4>
             Selecting a presentation display
           </h4>
+          <p>
+            The global tasks queued in these steps use the <a data-cite=
+            "!HTML#-interaction-task-source">user interaction task source</a>.
+          </p>
```

Comment:
Same as above, the "user interaction task source" is used in HTML for tasks that fire events in direct response to user interaction. Even though transient activation is required, the call to `start()` is programmatic.

Reply:
I used the "user interaction task source" since the user agent should (in nearly all cases) show some sort of user interface.  But it's definitely not a requirement.  So I will remove the reference here.